src/dtscalibration/dts_accessor_utils.py

```diff
@@ -29,24 +29,22 @@ def check_dims(
         The correct dimensions.
 
     Returns:
-    -------
+    --------
 
     """
     if not correct_dims:
         assert len(labels) > 1, "Define the correct dimensions"
 
         for li in labels[1:]:
-            assert (
-                ds[labels[0]].dims == ds[li].dims
-            ), li + " does not have the correct dimensions." " Should be " + str(
-                ds[labels[0]].dims
+            assert ds[labels[0]].dims == ds[li].dims, (
+                li + " does not have the correct dimensions."
```

Comment:
Assertions should be avoided in non-test code. It's better to raise an exception here.
See https://stackoverflow.com/a/40183030

Also the message can be a single line:
```py
    msg = f"{li} does not have the correct dimensions. Dims should be {ds[labels[0]].dims}"
```